dts/arm/silabs/xg21/efr32mg21a020f1024im32.dtsi

```diff
@@ -8,17 +8,15 @@
 #include <silabs/xg21/efr32mg21.dtsi>
 
 / {
-	sram0: memory@20000000 {
-		reg = <0x20000000 DT_SIZE_K(96)>;
-	};
-
 	soc {
 		compatible = "silabs,efr32mg21a020f1024im32", "silabs,efr32mg21", "silabs,efr32", "simple-bus";
-
-		flash-controller@50030000 {
-			flash0: flash@0 {
-				reg = <0 DT_SIZE_K(1024)>;
-			};
-		};
 	};
 };
+
+&flash0 {
+	reg = <0x08000000 DT_SIZE_K(1024)>;
```

Comment:
reg address should be 0x0 on xg21 (and xg22). Only xg23 and above have the flash at 0x08000000.

Reply:
Nice catch - My bad!